compose/xui-frontend.yml

```diff
@@ -34,10 +34,9 @@ services:
       HEALTH_CCD_DATA_API: http://ccd-data-store-api:4452/health
 
       APPINSIGHTS_INSTRUMENTATIONKEY: TESTVAR
-      IDAM_SECRET: xui_webapp_secrect
+      IDAM_SECRET: "${BEFTA_OAUTH2_CLIENT_SECRET_OF_XUIWEBAPP}"
       S2S_SECRET: "${BEFTA_S2S_CLIENT_SECRET_OF_XUI_WEBAPP}"
-
-      LAUNCH_DARKLY_CLIENT_ID: 1
+      LAUNCH_DARKLY_CLIENT_ID: 5de6610b23ce5408280f2268
```

Comment:
Sorry, I don't know enough about LaunchDarkly to know whether we should be changing this value or whether it is best controlled via a env var.  Please confirm with another team member, thanks.

Reply:
Shouldn't this be an environment variable? Putting a client id in a publicly accessible repo doesn't seem to be a good approach